unison-syntax/src/Unison/Parser/Ann.hs

```diff
@@ -10,11 +10,15 @@ data Ann
   = -- Used for things like Builtins which don't have a source position.
     Intrinsic -- { sig :: String, start :: L.Pos, end :: L.Pos }
   | External
+  | -- Indicates that the construct was generated from something at the given location.
+    -- E.g. record constructors are generated from their field definition.
+    GeneratedFrom Ann
```

Comment:
How does `GeneratedFrom` work? Where do we actually use it? (this one maybe easier on a call if you want). `Parser.Ann` is kind of a weird type to begin with. 😅

Reply:
i.e. What does it give us above and beyond the regular `Ann`?

Reply:
It's used for generated record field accessors, previously they were just given the annotation of the field they were generated from, which is a problem because we definitely shouldn't try to print them at that location.

Now they're explicitly tagged as not being in the source file, but rather being `GeneratedFrom` a position in the source file.

Reply:
Oh right, they're also the location to put the reformatted text back at.